e2e-scripts/sippy-e2e-sippy-e2e-setup-commands.sh

```diff
@@ -233,9 +233,9 @@ ${KUBECTL_CMD} -n postgres describe job sippy-load-job
 # We set +e to avoid the script aborting before we can retrieve logs.
 set +e
 # This takes under 3 minutes so 5 minutes (300 seconds) should be plenty.
-TIMEOUT=300s
```

Comment:
you took this one out and renamed it `SIPPY_LOAD_TIMEOUT` -- cool.
But looking at the test run, you have to define that somewhere in the scripts.